experimental/README.md

```diff
@@ -45,29 +45,26 @@ refined or entirely removed.
 
     This command downloads a test image and runs it in a container.
 
-### Get the Linux binary
-To download the latest experimental `docker` binary for Linux,
-use the following URLs:
+4. And verify the experimental flag is enabled:
 
-    https://experimental.docker.com/builds/Linux/i386/docker-latest.tgz
+        $ docker version -f '{{.Server.Experimental}}'
+        true
 
-    https://experimental.docker.com/builds/Linux/x86_64/docker-latest.tgz
```

Comment:
I removed this line because IIUC tgz is unlikely to contain `daemon.json` with the `experimental` flag.
Maybe we should mention RPM (and DEB, right?) here later.